apps/src/templates/progress/StageLock.jsx

```diff
@@ -0,0 +1,74 @@
+/**
```

Comment:
Potentially contentious decision here and with HiddenStageToggle to name a component identically to the component it is replacing (the difference being where they live).